store/tikv/coprocessor.go

```diff
@@ -491,12 +493,7 @@ const minLogCopTaskTime = 300 * time.Millisecond
 func (worker *copIteratorWorker) run(ctx context.Context) {
 	defer worker.wg.Done()
 	for task := range worker.taskCh {
-		respCh := worker.respChan
-		if respCh == nil {
-			respCh = task.respChan
-		}
-
-		worker.handleTask(ctx, task, respCh)
+		worker.handleTask(ctx, task, task.respChan)
```

Comment:
Why not PutToken()  here so that we don't need copResponseCollector?

Reply:
good catch, pull https://github.com/pingcap/tidb/pull/19244 request instead.